### PR TITLE
Serialize list of users rather than calling serialize on every user object

### PR DESCRIPTION
**Pull request by kjlundsgaard**
This should save queries on the release serializer and hopefully make the releases page a bit faster

@macqueen @ckj @MaxBittker

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]
This seems like an easy win.

File: src/sentry/api/serializers/models/release.py [hunk 1]
```diff
@@ -45,13 +45,14 @@ def get_users_for_commits(item_list):
         id__in=[ue.user_id for ue in user_emails],
         sentry_orgmember_set__organization_id=org_id
     )
-    users_by_id = dict((user.id, serialize(user)) for user in users)
+    users = serialize(list(users))
+    users_by_id = dict((user['id'], user) for user in users)
```
Comment (mattrobenolt) [inline src/sentry/api/serializers/models/release.py]: Also, ProTip:

```python
users_by_id = {user['id']: user for user in users}
```

dict comprehensions!

Comment (macqueen) [inline src/sentry/api/serializers/models/release.py]: you could also do this as a dictionary comprehension: `users_by_id = {user['id']: user for user in users}`

**Review by macqueen — APPROVED** [review]
🦄

File: src/sentry/api/serializers/models/release.py (hunk 1 above)
Comment (macqueen) [inline src/sentry/api/serializers/models/release.py]: woops had page open and didn't see matt's comment

Comment (kjlundsgaard) [inline src/sentry/api/serializers/models/release.py]: smh

**Pull request merged**